src/services/query.js

```diff
@@ -357,9 +357,22 @@ ngeo.Query.prototype.issue = function(map, object) {
 ngeo.Query.prototype.issueIdentifyFeaturesRequests_ = function(map, coordinate) {
   var sources = this.getQueryableSources_(map, false);
 
-  this.doGetFeatureInfoRequests_(sources.wms, coordinate, map);
-  this.doGetFeatureRequestsWithCoordinate_(sources.wfs, coordinate, map);
-  this.updatePendingState_();
+  // Launch requests only if there are queryable sources
+  var doneRequest = false;
+
+  if (Object.keys(/** @type {!Object} */ (sources.wms)).length) {
+    this.doGetFeatureInfoRequests_(sources.wms, coordinate, map);
+    doneRequest = true;
+  }
+
+  if (Object.keys(/** @type {!Object} */ (sources.wfs)).length) {
+    this.doGetFeatureRequestsWithCoordinate_(sources.wfs, coordinate, map);
+    doneRequest = true;
+  }
+
+  if (doneRequest) {
+    this.updatePendingState_();
```

Comment:
[updatePendingState](https://github.com/adube/ngeo/blob/cde75bab6989aa54f1415cb8c13f391831264b4a/src/services/query.js#L791-L799) checks if there are pending sources. Why did you have to introduce this variable `doneRequest`? Does it mean that there are still sources marked as `pending`? Which would be wrong, or not?